bin/index.js

```diff
@@ -15,6 +15,12 @@ const AdapterTestSuiteServer = require('../app/adapter-test-suite-server');
 const DEFAULT_SERVER_PORT = 5837;
 const DEFAULT_SECURE_SERVER_PORT = 5838;
 
+// =============================================================================
```

Comment:
Since the input argument is optional and if it exists, it's always going to be a string. So I will just add a validation in adapter-files.js instead of doing validation in this file.